code/modules/mob/living/simple_animal/simple_animal_vr.dm

```diff
@@ -92,21 +96,39 @@
 /mob/living/simple_animal/PunchTarget()
 	ai_log("vr/PunchTarget() [target_mob]", 3)
 
-	// For things we don't want to eat, call the sideways "parent" to do normal punching
-	if(!vore_active || !will_eat(target_mob))
+	// If we're not hungry, call the sideways "parent" to do normal punching
+	if(!vore_active)
 		return ..()
 
-	// If target is standing we might pounce and eat them
-	if(target_mob.canmove && prob(vore_pounce_chance))
-		target_mob.Weaken(5)
-		target_mob.visible_message("<span class='danger'>\the [src] pounces on \the [target_mob]!</span>!")
-
-	// If they're down or we can eat standing, do it
-	if(!target_mob.canmove || vore_standing_too)
+	// If target is standing we might pounce and knock them down instead of attacking
+	if(target_mob.canmove && prob(vore_pounce_chance) && !issilicon(target_mob) && (world.time > vore_pounce_cooldown)) //pouncing is worth doing, we want to pounce and are not in cooldown. So attempt a pounce!
+		if(will_eat(target_mob) && vore_standing_too) //creatures that can eat you on the spot don't care how healthy you are and get an autohit
+			return PounceTarget()
+
+		var/TargetHealthPercent = (target_mob.health/target_mob.maxHealth)*100
+		if (TargetHealthPercent <= vore_pounce_maxhealth) //check they're weak enough to bother pouncing
+			if(prob(vore_pounce_successrate - (vore_pounce_falloff * TargetHealthPercent))) // pounce success!
+				return PounceTarget()
+			else // pounce misses!
+				target_mob.visible_message("<span class='danger'>\the [src] attempts to pounce \the [target_mob] but misses!</span>!")
+				playsound(loc, 'sound/weapons/punchmiss.ogg', 25, 1, -1)
+				vore_pounce_cooldown = world.time + 20 SECONDS // don't attempt another pounce for a while
+				return // no you don't get to attack as well, you missed your chance
+
+	// We're not attempting a pounce, if they're down or we can eat standing, do it as long as they're edible. Otherwise, hit normally.
+	if(will_eat(target_mob) && (!target_mob.canmove || vore_standing_too))
 		return EatTarget()
 	else
 		return ..()
 
+/mob/living/simple_animal/proc/PounceTarget()
+	target_mob.Weaken(5)
+	target_mob.visible_message("<span class='danger'>\the [src] pounces on \the [target_mob]!</span>!")
```

Comment:
Cooldown in here. Where you actually pounce. Otherwise you might get:
a) the cooldown getting set when you don't actually pounce
b) pounces where you don't set the cooldown like you have above
It will be more consistent here.